tuning/sft_trainer.py

```diff
@@ -303,14 +303,24 @@ def train(
     logger.info("Packing is set to %s ", train_args.packing)
 
     data_preprocessing_time = time.time()
+    padding_free = ""
+    if attention_and_distributed_packing_config:
+        if attention_and_distributed_packing_config.padding_free:
+            padding_free = attention_and_distributed_packing_config.padding_free
```

Comment:
```suggestion
```

Reply:
Add this code in the [config class itself](https://github.com/foundation-model-stack/fms-hf-tuning/blob/main/tuning/config/acceleration_configs/attention_and_distributed_packing.py#L26C1-L35C52)


```python
@dataclass
class AttentionAndDistributedPackingConfig:
    ...
    def __post_init__(self):
         ....
     
    def is_padding_free(self:
           return self.padding_free is not None
```